packages/ui/package.json

```diff
@@ -11,6 +11,7 @@
     "lint": "eslint src",
     "lint:fix": "eslint --fix src",
     "format": "prettier --write --plugin-search-dir=. 'src/**/*.{ts*,js*,yml,svelte}' && eslint --fix src",
```

Comment:
@SasLord  Please just remove \' on format, it works fine on macos. And yaml seems to be not required.